src/mlpack/methods/ann/layer/sqnl_impl.hpp

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file methods/ann/layer/sqnl_impl.hpp
+ * @author Shaikh Yusuf Niaz
+ *
+ * Implementation of Square NonLinearity (SQNL) function as described by
+ * Wuraola, Adedamola and Patel, Nitish.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LAYER_SQNL_IMPL_HPP
+#define MLPACK_METHODS_ANN_LAYER_SQNL_IMPL_HPP
+
+// In case it hasn't yet been included.
+#include "sqnl.hpp"
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+template<typename InputDataType, typename OutputDataType>
+SQNL<InputDataType, OutputDataType>::SQNL() 
+{
+  // Nothing to do here
```

Comment:
```suggestion
  // Nothing to do here.
```

Missing stop at the end.